### ISSUE TITLE
general question regarding training and testing

### ISSUE THREAD

**Issue by seyeeet** [edgarschnfld/CADA-VAE-PyTorch#21]
Thanks for sharing your code. I am new to FSL and appreciate it if you can clarify this confusion for me.
In FSL we have support and query images.
Should we have all the supports and queries for each episode to be in one iteration?

**Comment by edgarschnfld** [comment]
Hi, there are different ways of conducting FSL. We are using the conceptually simplest setup: If you have K classes with N images each, and you train with S shots, then the training set for the classifier consists of K*S images, while the test set consists of K*(N-S) images. The classifier is only trained and test once.
